bigtable/writes/write_batch.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package writes
+
+// [START bigtable_writes_batch]
+import (
+	"context"
+	"encoding/binary"
+	"fmt"
+	"io"
+
+	"cloud.google.com/go/bigtable"
+)
+
+func writeBatch(w io.Writer, projectID, instanceID string, tableName string) error {
+	// projectID := "my-project-id"
+	// instanceID := "my-instance-id"
+	// tableName := "mobile-time-series"
+
+	ctx := context.Background()
+	client, err := bigtable.NewClient(ctx, projectID, instanceID)
+	if err != nil {
+		return fmt.Errorf("bigtable.NewAdminClient: %v", err)
+	}
+	tbl := client.Open(tableName)
+	columnFamilyName := "stats_summary"
+	timestamp := bigtable.Now()
+
+	var muts []*bigtable.Mutation
+
+	b := make([]byte, binary.MaxVarintLen64)
+	binary.PutVarint(b, int64(1))
```

Comment:
@billyjacobson unfortunately, this is Varint encoding that you are doing. I believe you want to do BigEndian encoding so
```go
binary.Write(buf, binary.BigEndian, count)
```
and here is how you can see the differences https://play.golang.org/p/vRTVE9a3gyT

Reply:
Oh thanks so much!